composer.json

```diff
@@ -41,7 +41,7 @@
         "symfony/polyfill-mbstring": "^1.28",
         "symfony/process": "^6.4.0 || ^7.0.0",
         "symfony/var-dumper": "^6.4.0 || ^7.0.0",
-        "thecodingmachine/safe": "^2.5",
+        "thecodingmachine/safe": "^2.5 || 3.0",
```

Comment:
I think we need to drop 2.5 here

Reply:
I couldn't remove 2.5 here because PHP-Scoper was still depending on it. PHP-Scoper no longer does now so this will be gone in the next update

Reply:
the recommendation is just to fix the constraint about major version 3, not to remove the 2.5 versions BC

Reply:
See
* https://github.com/box-project/box/pull/1450